agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/CallDepthThreadLocalMap.java

```diff
@@ -31,24 +40,36 @@ protected ThreadLocalDepth computeValue(Class<?> type) {
         }
       };
 
+  public static Depth getCallDepth(final Class<?> k) {
+    return TLS.get(k).get();
+  }
+
   public static int incrementCallDepth(final Class<?> k) {
-    return TLS.get(k).get().increment();
+    return TLS.get(k).get().getAndIncrement();
+  }
+
+  public static int decrementCallDepth(final Class<?> k) {
```

Comment:
i think this is not used?

Reply:
I have added it mostly for symmetry and it is used from tests